venues/ICLR.cc/2018/Conference/python/programchairs/get-review-info.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+
+"""
+Output review information for all papers to a csv file.
+Paper Num, Title, Area Chair, reviews complete/total, Reviewer1, rating, confidence, Reviewer2...
+
+"""
+
+## Import statements
+import argparse
+from openreview import *
+import csv
+import sys
+sys.path.insert(0, "./..")
+import config
+
+## Handle the arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+parser.add_argument('-f', '--file',help="specify an output file otherwise defaults to info.csv")
+parser.add_argument('--ac',help="specify area chair")
```

Comment:
I think this parameter is not being used

Reply:
It is on line 119.  It allows you to output information of only one Area Chair.  Not necessary, but output seemed less overwhelming.